src/ray/core_worker/BUILD.bazel

```diff
@@ -62,6 +61,21 @@ ray_cc_library(
     ],
 )
 
+ray_cc_library(
+    name = "shutdown_coordinator",
+    srcs = [
+        "shutdown_coordinator.cc",
+    ],
+    hdrs = [
+        "shutdown_coordinator.h",
+    ],
+    visibility = [":__subpackages__"],
+    deps = [
+        ":common",
```

Comment:
common only contains a few seemingly-irrelevant utilities: https://github.com/ray-project/ray/blob/d92a3234654c3a102292548cc15be3b64ef5da0f/src/ray/core_worker/common.cc

it looks like the only thing we're pulling it in for is a pretty pointless type alias. let's directly depend on the protobuf target that's required instead of this

Reply:
Done